### ISSUE TITLE
dropped_vnode_requests_total missing in riak-admin status output

### ISSUE THREAD

**Issue by runesl** [basho/riak_kv#740]
dropped_vnode_requests_total is in /stats output, but not in the output from riak-admin status when calling from command line.

Observed on 1.4.2 and 2.0.0pre5

In addition, it would be nice if this stat also had a 1 minute-version "dropped_vnode_requests"  like most of the other total counters do.

**Comment by slfritchie** [comment]
NOTE: There are a number of other stats that are missing from "riak-admin status" output, alas, that do indeed appear in the HTTP server's /stats JSON blob, e.g. riak_kv_vnodeq_max.

**Comment by russelldb** [comment]
@slfritchie list 'em and I'll fix 'em :D

**Comment by slfritchie** [comment]
Here's what I've found with brute-force sed & diff, run on today's `develop` branch of Riak.

```
diff --git a/tmp/stats.riak-admin-status b/tmp/stats.slash-stats
index 3a508ed..4ba85c7 100644
--- a/tmp/stats.riak-admin-status
+++ b/tmp/stats.GET-slash-stats
@@ -1,10 +1,14 @@
-1-minute stats for 'riak@127.0.0.1' says:
+GET /stats says:
 asn1_version
 basho_stats_version
 bitcask_version
 cluster_info_version
 compiler_version
 connected_nodes
+converge_delay_last
+converge_delay_max
+converge_delay_mean
+converge_delay_min
 coord_redirs_total
 counter_actor_counts_100
 counter_actor_counts_95
@@ -17,12 +21,16 @@ cpu_avg5
 cpu_nprocs
 crypto_version
 disk
+dropped_vnode_requests_total
 eleveldb_version
 erlang_js_version
 erlydtl_version
 executing_mappers
 goldrush_version
+gossip_received
+handoff_timeouts
 ibrowse_version
+ignored_gossip_total
 index_fsm_active
 index_fsm_create
 index_fsm_create_error
@@ -213,22 +221,42 @@ read_repairs_map_total
 read_repairs_set
 read_repairs_set_total
 read_repairs_total
+rebalance_delay_last
+rebalance_delay_max
+rebalance_delay_mean
+rebalance_delay_min
+rejected_handoffs
 riak_api_version
 riak_auth_mods_version
 riak_control_version
+riak_core_stat_ts
 riak_core_version
 riak_dt_version
 riak_kv_stat_ts
 riak_kv_version
+riak_kv_vnodeq_max
+riak_kv_vnodeq_mean
+riak_kv_vnodeq_median
+riak_kv_vnodeq_min
+riak_kv_vnodeq_total
+riak_kv_vnodes_running
 riak_pb_version
 riak_pipe_stat_ts
 riak_pipe_version
+riak_pipe_vnodeq_max
+riak_pipe_vnodeq_mean
+riak_pipe_vnodeq_median
+riak_pipe_vnodeq_min
+riak_pipe_vnodeq_total
+riak_pipe_vnodes_running
 riak_search_version
 riak_sysmon_version
 ring_creation_size
 ring_members
 ring_num_partitions
 ring_ownership
+rings_reconciled
+rings_reconciled_total
 runtime_tools_version
 sasl_version
 set_actor_counts_100
```

**Comment by slfritchie** [comment]
Given the number of differences ... how quick is it to avoid a 1-by-1 inclusion in "riak-admin status"?

**Comment by russelldb** [comment]
`riak_kv_status:statistics/0` returns only `riak_kv_stat:get_stats/0` but `/stats` returns the union of `riak_kv_stats:get_stats/0` and `riak_core_stat:get_stats/0`. The fix is pretty easy and on its way.

**Comment by shino** [comment]
Related issue: https://github.com/basho/riak_core/issues/474

**Comment by russelldb** [comment]
https://github.com/basho/riak_kv/pull/828

**Comment by jtuple** [comment]
Resolved in #828, closing.
